music21/languageExcerpts/instrumentLookup.py

```diff
@@ -208,12 +216,15 @@
     'ocarina': 'Ocarina',
     'organ': 'Organ',
     'pan flute': 'PanFlute',
+    'percussion': 'Percussion',
     'piano': 'Piano',
     'piccolo': 'Piccolo',
     'pipe organ': 'PipeOrgan',
+    'pitched percussion': 'Pitched Percussion',
```

Comment:
no space. here's a great example of why we should move over to class names as values, because this would have been caught.